Sources/Mock.swift

```diff
@@ -156,3 +164,11 @@ public struct Mock: Equatable {
         return lhs.url.absoluteString == rhs.url.absoluteString && lhsHTTPMethods == rhsHTTPMethods
     }
 }
+
+private extension URL {
```

Comment:
Doesn't mocker has URL extensions already? If so, we could combine them in the same file.

Reply:
Turned out it doesn't have one yet. To keep the library simple and small I'm keeping it in this file.